ui/raidboss/data/06-ew/raid/p7s.ts

```diff
@@ -101,6 +102,41 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.startsUsing({ id: '7823', source: 'Agdistis', capture: false }),
       response: Responses.goRight(),
     },
+    {
+      id: 'P7S Roots of Attis 3',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '780E', source: 'Agdistis', capture: false }),
+      condition: (data) => data.secondRoots === false,
+      infoText: (_data, _matches, output) => output.baitSoon!(),
+      outputStrings: {
+        baitSoon: {
+          en: 'Bait on Empty Platform Soon',
+        },
+      },
+    },
+    {
+      id: 'P7S Roots of Attis 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '780E', source: 'Agdistis', capture: false }),
+      condition: (data) => data.secondRoots,
+      infoText: (_data, _matches, output) => output.separateHealerGroups!(),
+      run: (data) => data.secondRoots = false,
+      outputStrings: {
+        separateHealerGroups: {
+          en: 'Healer Group Platforms',
+        },
+      },
+    },
+    {
+      // First breaks north bridge for upcoming South Knockback Spreads
+      // Second breaks remaining bridges, Separate Healer Groups
+      // Third breaks all bridges, Bait on Empty Platform
+      id: 'P7S Roots of Attis 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '780E', source: 'Agdistis', capture: false }),
+      condition: (data) => data.secondRoots === undefined,
```

Comment:
Sorry for the nit, but I think it makes it more confusing to follow when you check both `data.secondRoots` and `data.secondRoots === undefined`.  Personally, I think if you have an optional boolean, then undefined/false should be equivalent.  Could you just make this a roots counter and increment when you see roots? I think this would make it easier to read.

Reply:
Ok, I've updated it to be a number.